src/cpp/runner/runnerapp.cpp

```diff
@@ -802,20 +799,14 @@ class RunnerApp::Private : public QObject
 	}
 };
 
-RunnerApp::RunnerApp(QObject *parent) :
-	QObject(parent)
-{
-	d = new Private(this);
+RunnerApp::RunnerApp() {
+    d = std::make_unique<Private>(this);
 }
 
-RunnerApp::~RunnerApp()
-{
-	delete d;
-}
+RunnerApp::~RunnerApp() = default;
```

Comment:
I didn't even know you could do that! maybe useful for libs. in our case I'd say we don't need to specify this at all and can just remove the destructor from .cpp & .h when we only want the default.

Reply:
we need to keep destructor def due to :
```
function `RunnerAppMain::app_quit(int)':
         runnermain.cpp:41: reference to `RunnerApp::~RunnerApp()'
```